compiler/rustc_target/src/spec/targets/m68k_unknown_none_elf.rs

```diff
@@ -0,0 +1,30 @@
+use crate::abi::Endian;
+use crate::spec::{CodeModel, PanicStrategy, Target, TargetOptions};
+
+pub(crate) fn target() -> Target {
+    let options = TargetOptions {
+        cpu: "M68010".into(),
+        max_atomic_width: None,
+        endian: Endian::Big,
+        // LLD currently does not have support for M68k
+        linker: Some("m68k-linux-gnu-ld".into()),
+        panic_strategy: PanicStrategy::Abort,
+        code_model: Some(CodeModel::Medium),
+        has_rpath: false,
+        ..Default::default()
```

Comment:
So, we don't necessarily need to _use_ it right now, but can you specify the `llvm_floatabi` field and attach a comment so that we know the _intended_ float ABI for this target, even if we're just repeating the default `None`? I'm guessing soft float?
```suggestion
        // should be soft-float
        llvm_floatabi: None,
        ..Default::default()
```